feat(changelog): object-storage-added-access-logs-now-available 2023-09-18 (#2152)
scaleway/docs-content: changelog/september2023/2023-09-18-object-storage-added-access-logs-now-available.mdx

---

> title: Access logs now available in Paris Region

Reviewer: ```suggestion
title: Access logs now available in the Paris Region
```

status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-09-18
category: storage
product: object-storage
---

> We are thrilled to announce a significant enhancement to Scaleway Object Storage that will provide you with even greater visibility and control over your data. Starting today, access logs are now available in Cockpit for Paris Region!

Reviewer: ```suggestion
We are thrilled to announce a significant enhancement to Scaleway Object Storage that will provide you with even greater visibility and control over your data. Starting today, access logs are now available in Cockpit for the Paris Region!
```

> We are thrilled to announce a significant enhancement to Scaleway Object Storage that will provide you with even greater visibility and control over your data. Starting today, access logs are now available in Cockpit for the Paris Region!

Reviewer: ```suggestion
Starting today, access logs are now available in Cockpit for the Paris Region! This enhancement will provide you with even greater visibility and control over your data.
```
